Examples.fsh

```diff
@@ -55,7 +80,11 @@ InstanceOf: CancerPatient
 * id = "mCODEPatientExample02"
 * meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/CancerPatient"
 // The following lines must use the slice names, not the defining URLs
-* extension[race].extension[ombCategory].valueCoding = OmbCat#2186-5 "Not Hispanic or Latino"
+* extension[race].url = "http://hl7.org/fhir/us/core/StructureDefinition/us-core-race"
+* extension[race].extension[ombCategory].url = "ombCategory"
+* extension[race].extension[ombCategory].valueCoding = USCoreRace#1002-5 "American Indian or Alaska Native"
+* extension[race].extension[text].url = "text"
+* extension[race].extension[text].valueString = "White"
```

Comment:
Does "White" comport with "American Indian or Alaska Native"?

Reply:
Oops you're right - forgot to change the text.